build_docs.sh

```diff
@@ -1,14 +1,11 @@
 #!/bin/bash
+gem install jazzy
+echo -e "Copying MSAL public files"
```

Comment:
Should this be done only for master branch? I don't think we want to regenerate docs on each PR.

Reply:
Yes, it is only done for master since it will be executed in spm-framework.yml, which is our final release pipeline.